### ISSUE TITLE
Change error message into instruction

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1228]
### Description

Error message is not suggestive
<img width="364" alt="264321033-405f2006-1e69-4c4e-a668-eb5b8b6dcf1a" src="https://github.com/cryptpad/cryptpad/assets/95225431/d9992aec-9816-4980-82bd-3eda4a6d8b40">

### Steps to reproduce

1. Go to team's administration 
2. Try to change the name with the name it currently has

### Relevant logs

```shell
.
```

### Expected behavior

The error message should be changed into something more suggestive (should describe what went wrong)

### What opreating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox, Google Chrome

### Extensions

.

### Version

5.4.1

**Comment by dariiing** [comment]
<img width="341" alt="image" src="https://github.com/cryptpad/cryptpad/assets/95225431/2042c838-75d5-44ab-8420-7514f0622d1e">
